src/main/java/com/jcraft/jsch/JSch.java

```diff
@@ -608,14 +608,10 @@ public static String getConfig(String key) {
    *
    * @param newconf configurations
    */
-  public static void setConfig(Hashtable<String, String> newconf) {
+  public static void setConfig(Map<String, String> newconf) {
     synchronized (config) {
-      for (Enumeration<String> e = newconf.keys(); e.hasMoreElements();) {
-        String newkey = e.nextElement();
-        String key =
-            (newkey.equals("PubkeyAcceptedKeyTypes") ? "PubkeyAcceptedAlgorithms" : newkey);
-        config.put(key, newconf.get(newkey));
-      }
+      newconf.forEach((key, value) -> config
+          .put(key.equals("PubkeyAcceptedKeyTypes") ? "PubkeyAcceptedAlgorithms" : key, value));
```

Comment:
I would rather just see a traditional for loop over an `newconf.entrySet()` instead of the `.forEach()` lambda.